draft-westerlund-tsvwg-dtls-over-sctp-bis.md

```diff
@@ -196,6 +191,8 @@ This update that replaces RFC 6083 defines the following changes:
 * Requires that SCTP-AUTH is applied to all SCTP Chunks that can be
      authenticated.
 
+* Require support of partial delivery of user messages {{RFC6458}}.
+
```

Comment:
Not sure we need a reference to RFC6458 here. I would just remove it.

Reply:
Agreed, a result of writing changes in the wrong order.